beamer/health/check.py

```diff
@@ -192,27 +210,54 @@ def fetch_events() -> ChainEventMap:
     return cast(ChainEventMap, events)
 
 
-def make_web3(rpc: str) -> Web3:
-    web3 = Web3(Web3.HTTPProvider(rpc, request_kwargs=dict(timeout=5)))
-    web3.middleware_onion.inject(geth_poa_middleware, layer=0)
-    return web3
+def get_transfer_token_symbol(transfer: Transfer, token_deployments: TokenMap) -> str | None:
+    for token_symbol, deployments in token_deployments.items():
+        if any(
+            transfer["created"].source_token_address.lower() == address.lower()
+            and transfer["created"].chain_id == int(chain_id)
```

Comment:
You can move both `transfer["created"].source_token_address.lower()` and ` transfer["created"].chain_id` out of the loop and into local variables since they are invariant with respect to the loop and thereby save unnecessary operations. As a bonus, the code inside `any(...)` will be shorter and more readable.